src/Admin.php

```diff
@@ -32,6 +32,9 @@ public static function init() {
     // Exposes SVG images in media library.
     add_filter('wp_prepare_attachment_for_js', __CLASS__ . '::wp_prepare_attachment_for_js');
     add_action('admin_head', __CLASS__ . '::admin_head');
+
+    // Stops MCE editor setting unwanted rel attribute values for links that open in new window.
```

Comment:
```diff
-    // Stops MCE editor setting unwanted rel attribute values for links that open in new window.
+    // Prevents TinyMCE from setting unwanted rel attribute for links opening in new window.
```

Let's also move this new filter in between the sample_permalink and the SVG images, so that the filters are registered in context and logical execution order.